src/components/ui/Input/index.tsx

```diff
@@ -54,9 +49,11 @@ const Input: React.FC<InputProps> = ({
       data-testid={id}
       leftIcon={icon}
       keyboardType={keyboard}
-      inputContainerStyle={[styles.inputContainer, inputContainerStyle]}
-      inputStyle={[styles.input, inputStyle]}
+      inputContainerStyle={styles.inputContainer}
+      inputStyle={editable ? styles.input : styles.uneditableInput}
```

Comment:
Would also be good to update the styles to take in an argument and then change the input style accordingly